uptane-standard.md

```diff
@@ -572,8 +576,8 @@ Unless stated otherwise, all files SHALL be written to repositories in accordanc
 
 For example:
 
-* The version number of the Snapshot metadata file is 61, and its filename in the Timestamp metadata is "snapshot.json". The filename on the repository will be "61.snapshot.json".
-* There is an image with the filename "acme_firmware.bin" specified in the Targets metadata, with a SHA3-256 of "aaaa" and a SHA-512/224 of "bbbb". It will have two filenames on the repository: "aaaa.acme_firmware.bin" and "bbbb.acme_firmware.bin".
+* The version number of the Snapshot metadata file is 61, and its filename in the Timestamp metadata is "snapshot.json." The filename on the repository will be "61.snapshot.json."
+* There is an image with the filename "acme_firmware.bin" specified in the Targets metadata, with a SHA3-256 of "aaaa" and a SHA-512/224 of "bbbb". It will have two filenames on the repository: "aaaa.acme_firmware.bin" and "bbbb.acme_firmware.bin."
```

Comment:
Can we maybe use something other than quotation marks here? If we follow the style advice of putting periods inside quotes, then this gets really confusing. It's super unclear whether the periods are meant to be part of the name or not. Besides, these aren't quotes; we're just trying to delineate the name from the surrounding text. Could we use backticks (`) or something else?

Reply:
I agree that the question marks are distracting. I think we could use italics instead.

Reply:
Italics sound fine to me. If that doesn't look clear enough, Ira's suggested angle brackets (<>) would also be fine with me.

Reply:
Ok. Let me go through the document and see if I can replace all the quotes used in this manner with italics. If it looks weird, we can always try something else.

Reply:
fixed in latest commit